Lib/idlelib/editor.py

```diff
@@ -1203,10 +1203,7 @@ def load_extension(self, name):
             self.apply_bindings(keydefs)
             for vevent in keydefs:
                 methodname = vevent.replace("-", "_")
-                while methodname[:1] == '<':
-                    methodname = methodname[1:]
-                while methodname[-1:] == '>':
-                    methodname = methodname[:-1]
+                methodname = methodname.lstrip('<').rstrip('>')
                 methodname = methodname + "_event"
```

Comment:
Actually, the original 6 lines can be replaced with 1:
`methodname = methodname.lstrip('<').rstrip('>')